### PR TITLE
Update dependancies

### PR DESCRIPTION
**Pull request by crutchcorn**
This merge requests updates to babel@8, webpack@4, eslint@5

### CODE REVIEW

**Comment by shershen08** [comment]
Thanks for PR @crutchcorn 
I'll check that today and get back with the feedback

File: dist/vue-masonry-plugin-umd.js [hunk 1]
```diff
@@ -3,4 +3,38 @@
...
+/*!
+ * Outlayer v2.1.1
+ * the brains and guts of a layout library
+ * MIT license
+ */
+/*!
+ * Outlayer v2.1.1
```
Comment (crutchcorn) [inline dist/vue-masonry-plugin-umd.js]: Oh dear, it seems that the uglify config that was migrated doesn't remove comments for builds. I'll look into how to fix that

**Comment by crutchcorn** [comment]
Two questions I had (that might be potentially out-of-scope for this PR:
1) Should the project build both files with one command? It does not do so currently
2) Is there a need/want for pushing `dist` to the repo. There is a `postinstall` hook for `npm` and `yarn` that could handle the build when the user `npm install`/`yarn install`s. I understand there seems to be wanted `bower` support, but I know their site defers to `yarn` for the front-end

**Comment by shershen08** [comment]
@crutchcorn about 2nd part of your question 

> Is there a need/want for pushing dist to the repo

I've added /dist to a repo to be able to access it directly in browser environment - https://github.com/shershen08/vue-masonry#usage-directly-in-the-browser

**Comment by shershen08** [comment]
about 

> Should the project build both files with one command? It does not do so currently

true, there was no need in this, since it's just a plugin library - I am not developing/building it on daily basis

**Comment by crutchcorn** [comment]
@shershen08 was there more feedback for this PR or do you think it should be ready to go?

**Pull request merged**